lib/modules/dosomething/dosomething_kudos/dosomething_kudos.module

```diff
@@ -325,12 +325,10 @@ function dosomething_kudos_get_kudos_data_for_fid($fid = NULL, $total_rogue_kudo
     return [];
   }
 
-  if ($total_rogue_kudos === 0) {
-    $total_kudos = 0;
-  } elseif ($total_rogue_kudos) {
-    $total_kudos = $total_rogue_kudos;
-  } else {
+  if(!isset($total_rogue_kudos)) {
```

Comment:
Not a merge blocker, but I think we'd have an easier time thinking about this if we always split code paths based on the feature flag, rather than based on whether a param exists or not.